source/macos/AdaptiveCards/AdaptiveCards-bridge/BridgeUtils/BridgeTextUtils.h

```diff
@@ -12,6 +12,6 @@
 + (ACSRichTextElementProperties * _Nonnull)convertTextBlockToRichTextElementProperties:(ACSTextBlock * _Nonnull)textBlock;
 + (ACSRichTextElementProperties * _Nonnull)convertTextRunToRichTextElementProperties:(ACSTextRun * _Nonnull)textRun;
 + (ACSRichTextElementProperties * _Nonnull)convertFactToRichTextElementProperties:(ACSFact * _Nonnull)fact;
-+ (ACSMarkdownParserResult * _Nonnull)processTextFromFact:(ACSFact * _Nullable)fact hostConfig:(ACSHostConfig * _Nonnull)config;
++ (ACSMarkdownParserResult * _Nonnull)processTextFromFact:(ACSFact * _Nullable)fact hostConfig:(ACSHostConfig * _Nonnull)config isTitleField: (BOOL)isTitleField;
```

Comment:
👍

Reply:
Maybe rename the param as `isTitle`. 
Contextually here, only whether it is a title matters here.